examples/Yolo-object-detection/client/get_data.py

```diff
@@ -0,0 +1,87 @@
+import os
+import glob
+import zipfile
+import requests
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+def download_config():
+    url = "https://storage.googleapis.com/public-scaleout/Yolo-object-detection/1.zip"
+    response = requests.get(url)
+    os.makedirs(dir_path + "/data", exist_ok=True)
+    with open(dir_path + "/data/1.zip", "wb") as f:
+        f.write(response.content)
+    zip_file_path = dir_path + "/data/1.zip"
+    extract_to_path = dir_path + "/data/1/"
+
+    os.makedirs(extract_to_path, exist_ok=True)
+
+    with zipfile.ZipFile(zip_file_path, "r") as zip_ref:
+        zip_ref.extractall(extract_to_path)
+def download_blob():
+    url = "https://storage.googleapis.com/public-scaleout/Yolo-object-detection/data.zip"
+    response = requests.get(url)
+
+    with open(dir_path + "/data/1/data.zip", "wb") as f:
+        f.write(response.content)
+    zip_file_path = dir_path + "/data/1/data.zip"
+    extract_to_path = dir_path + "/data"
+
+    os.makedirs(extract_to_path, exist_ok=True)
+
+    with zipfile.ZipFile(zip_file_path, "r") as zip_ref:
+        zip_ref.extractall(extract_to_path)
+
+def train_test_creation(dataset=dir_path + "/data/1/data/", outdir=dir_path + "/data/1/", test_percentage=10):
```

Comment:
Do we have the capability here to split the dataset into multiple different clients? We do not longer have an environment variable controlling the number of splits?